backend/uclapi/dashboard/api_applications.py

```diff
@@ -202,6 +202,13 @@ def delete_app(request):
     else:
         app = apps[0]
         app.deleted = True
+        webhook = app.webhook
+        webhook.url = ""
```

Comment:
Since `url`, `siteid`, `roomid` and `contact` are all nullable, can we make these `= None` instead? I think it'll be neater.

```suggestion
        webhook.url = None
```

Reply:
@ChristopherHammond13 url is not nullable, everything else can be made none though yes.